packages/main/src/RPA/MSGraph.py

```diff
@@ -190,14 +190,43 @@ def get_drive_instance(
         else:
             return storage.get_default_drive()
 
-    def _parse_drive_param(self, drive: DriveType) -> drive_module.Drive:
+    def _combine_drive_params(
+        self, drive_id: Optional[str] = None, drive: Optional[DriveType] = None
+    ) -> DriveType:
+        """Function to be removed once ``drive_id`` is fully deprecated."""
+        if drive_id is not None:
+            raise DeprecationWarning(
```

Comment:
commented already in the Slack: as this is a new library, it is fine for me to just change parameters and make a breaking change release

Reply:
Okay, in that case I'll update everything to just remove `drive_id`